documentation/modules/exploit/windows/smb/psexec.md

```diff
@@ -1,16 +1,17 @@
-PSexec is one of the most popular exploits against Microsoft Windows. It is a great way to test password security and demonstrate how a
+PsExec is one of the most popular exploits against Microsoft Windows. It is a great way to test password security and demonstrate how a
 stolen password could lead to a complete compromise of an entire corporate network.
 
 ## Vulnerable Application
 
-To be able to use exploit/windows/smb/psexec:
+To be able to use `exploit/windows/smb/psexec`:
 
-1. You must have a valid username/password.
-2. The firewall must allow SMB traffic.
-3. The target must use SMBv1.
-4. The remote Windows machine's network security policy must allow it. If you see
-  [one of these errors](https://github.com/rapid7/metasploit-framework/wiki/What-does-my-Rex%3A%3AProto%3A%3ASMB-Error-mean%3F), then the
-  Windows machine does not allow it.
+1. A valid username and password must be set.
+1. The firewall must allow SMB traffic.
+1. The remote Windows machine's network security policy must allow it.
+    * If the specified account is a local Administrator and the target is Windows Vista or newer, then "Remote UAC" must be disabled (the
+      `DWORD` value `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System\LocalAccountTokenFilterPolicy` must be 1).
+      See [Pass-the-Hash Is Dead: Long Live LocalAccountTokenFilterPolicy][1] for more information. Without this setting, the server will
+      respond with `STATUS_ACCESS_DENIED` and PsExec will fail.
```

Comment:
Changes here look good, but you undid the numbered count. It should be `1.`, `2.`, etc. I'll fix this in a ninja edit before I merge this PR in.

Reply:
Like the new description here for the LocalAccountTokenFilterPolicy setting here, very nice overview without diving into too much of the specifics, good job!

Reply:
Woops according to @smcintyre-r7 this is a dcoumented feature of markdown. Closing this as this is not an issue and therefore we can merge this in as is.